### PR TITLE
Review Page Prep for Final Voting Page UI tweaks

### PR DESCRIPTION
**Pull request by Raemon**
This makes a number of UI improvements on the ReviewVotingPage. The primary goal was to clarify that posts need at least 1 review to make it to the Final Voting Page.

Emphasizing reviews:

– Changes the text of the instruction area 
– Shows the total Reviewed Posts more prominently than "nominated posts", makes it green, and gives it a tooltip (which also states "posts need at least 1 review"
– sets the default sorting to "needsReview"

Make it easier to sort by lastCommentedAt:
– Makes it possible to sort at all by lastCommentedAt
– makes it so when you click on a ReviewVoteTableRow, it turns the comment box grey so you can see that you've now read the comments there
– Changes the color of the strong/medium review votes from green to grey, so that it's easier to scan the list for unread comments.

I think it's plausible the default sorting should be by lastCommentedAt instead of needsReview, but weakly guessed the latter was better.

image

### CODE REVIEW

**Comment by Raemon** [comment]
Oh blarg, I forgot I need to actually set the markAsRead function when clicking on ReviewVoteTableRow. Will add that tomorrow.

**Comment by Raemon** [comment]
Added the "mark as read" feature to the ReviewVoteTableRow. Ready for review now.

File: packages/lesswrong/components/review/ReviewVoteTableRow.tsx [hunk 1]
```diff
@@ -177,7 +186,7 @@ const ReviewVoteTableRow = (
         <PostsItemComments
           small={false}
           commentCount={postGetCommentCount(post)}
-          unreadComments={post.lastVisitedAt < post.lastCommentedAt}
+          unreadComments={(post.lastVisitedAt < (markedVisitedAt || post.lastCommentedAt))}
```
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVoteTableRow.tsx:189]:
```suggestion
          unreadComments={(markedVisitedAt || post.lastVisitedAt) < post.lastCommentedAt}
```
The code works as it was written, but is not logically what you wanted.

**Review by jpaddison3 — APPROVED** [review]
ASTLAC

File: packages/lesswrong/components/review/ReviewVotingPage.tsx [hunk 2]
```diff
@@ -494,14 +503,17 @@ const ReviewVotingPage = ({classes}: {
...
+                </LWTooltip> 
+                ({sortedPosts.length} Nominated)
+              </div>
+            }
             
             {(postsLoading || loading) && <Loading/>}
             
             {/* Turned off for the Preliminary Voting phase */}
-            {getReviewPhase() === "VOTING" && <>
+            {/* {getReviewPhase() === "VOTING" && <>
```
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingPage.tsx:516]: Why though?

**Pull request merged**